quickstart/101-key-vault-key/variables.tf

```diff
@@ -0,0 +1,77 @@
+locals {
+  current_user_id = coalesce(var.msi_id, data.azurerm_client_config.current.object_id)
+}
+
+variable "resource_group_location" {
+  type        = string
+  description = "Location for all resources."
+  default     = "eastus"
+}
+
+variable "resource_group_name_prefix" {
+  type        = string
+  description = "Prefix of the resource group name that's combined with a random ID so name is unique in your Azure subscription."
+  default     = "rg"
+}
+
+variable "vault_name" {
+  type        = string
+  description = "The name of the key vault to be created. The value will be randomly generated if blank."
+  default     = ""
+}
+
+variable "key_name" {
+  type        = string
+  description = "The name of the key to be created. The value will be randomly generated if blank."
+  default     = ""
+}
+
+variable "sku_name" {
+  type        = string
+  description = "The SKU of the vault to be created."
+  default     = "standard"
+  validation {
+    condition     = contains(["standard", "premium"], var.sku_name)
+    error_message = "The sku_name must be one of the following: standard, premium."
+  }
+}
+
+variable "key_permissions" {
+  type        = list(string)
+  description = "List of key permissions."
+  default     = ["List", "Create", "Delete", "Get", "Purge", "Recover", "Update", "GetRotationPolicy", "SetRotationPolicy"]
+}
+
+variable "secret_permissions" {
```

Comment:
This variable should be included in the readme.md.

Reply:
I added the following vars to readme: secret_permissions, key_permissions